\doxysection{Timeout\+Error Class Reference}
\hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1_timeout_error}{}\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1_timeout_error}\index{TimeoutError@{TimeoutError}}


Inheritance diagram for Timeout\+Error\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=302pt]{d7/daa/classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1_timeout_error__inherit__graph}
\end{center}
\end{figure}


Collaboration diagram for Timeout\+Error\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=152pt]{d2/dcb/classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1_timeout_error__coll__graph}
\end{center}
\end{figure}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Raised when a socket timeout error occurs.

Catching this error will catch both :exc:`ReadTimeoutErrors
<ReadTimeoutError>` and :exc:`ConnectTimeoutErrors <ConnectTimeoutError>`.
\end{DoxyVerb}
 

Definition at line \mbox{\hyperlink{__vendor_2urllib3_2exceptions_8py_source_l00110}{110}} of file \mbox{\hyperlink{__vendor_2urllib3_2exceptions_8py_source}{exceptions.\+py}}.



The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
E\+:/pqc-\/engineering-\/ssec-\/23/venv/lib/python3.\+12/site-\/packages/pip/\+\_\+vendor/urllib3/\mbox{\hyperlink{__vendor_2urllib3_2exceptions_8py}{exceptions.\+py}}\end{DoxyCompactItemize}
